Package.swift

```diff
@@ -7,23 +7,39 @@ let package = Package(
     products: [
         .executable(
             name: "nio-apns-example",
-            targets: ["NIOAPNSExample"]),
+            targets: ["NIOAPNSExample"]
+        ),
         .library(
             name: "NIOAPNS",
-            targets: ["NIOAPNS"]),
+            targets: ["NIOAPNS"]
+        ),
     ],
     dependencies: [
-      .package(url: "https://github.com/moritzsternemann/nio-h2", .upToNextMinor(from: "0.1.0")),
+        .package(
+            url: "https://github.com/moritzsternemann/nio-h2",
+            .upToNextMinor(from: "0.1.0")
+        ),
+        .package(
+            url: "https://github.com/IBM-Swift/OpenSSL",
+            .upToNextMinor(from: "2.2.1")
+        ),
+        .package(
+            url: "https://github.com/apple/swift-package-manager",
```

Comment:
This is a huge dependency to be bringing in. Is there any way to make this internal?

Reply:
It's only used for the example CLI. This was just to get things up and running.. Do you have any suggestions or should I just implement this myself?

Reply:
Maybe just copy / paste (with attribution) the `ArgumentParser` class from SPM into the example project? Seems like that's the only thing you're using. Vapor also has a [console / command](https://docs.vapor.codes/3.0/command/overview/) framework, but there are no tags for the NIO 2 compatible code yet and honestly even that is probably too big a dependency for this use case.

Reply:
Hmm, I think I'll just parse the arguments on my own.. It's just an example for the APNS client and not for ArgumentParser anyways 😄